ft_prepare_leadfield.m

```diff
@@ -47,6 +47,22 @@
 %                         whether the lower rank leadfield is projected back onto the original
 %                         linear subspace, or not.
 %
+% Depending on the type of headmodel, some additional options may be
+% specified. 
+%
+% For OPENMEEG based headmodels:
```

Comment:
the option should be called cfg.openmeeg rather than cfg.om, for consistency with ft_prepare_headmodel